samples/typescript/21.react-media-template/src/utils/AzureMediaPlayer.ts

```diff
@@ -372,9 +373,6 @@ export class AzureMediaPlayer extends EventTarget {
                 // this._applySkipTo(false);
                 break;
             case PlayerEvent.ended:
-                this.load();
```

Comment:
I haven't checked the JS version of the sample but these changes may need to be replicated there as well.